expyfun/_input_controllers.py

```diff
@@ -84,6 +99,17 @@ def _on_pyglet_keypress(self, symbol, modifiers, emulated=False):
             this_key = this_key.lstrip('_').lstrip('NUM_')
         self._keyboard_buffer.append((this_key, key_time))
 
+    def _on_pyglet_keyrelease(self, symbol, modifiers, emulated=False):
```

Comment:
But `self._keyboard_buffer_releases` is just a list you created... When you do `self.win.dispatch_events()` you're still going to get the same information from the window manager (which is press times, not release times) regardless of the name of the list you stick the values into.